1. Вы провели эксперимент c упрощением формы заказа в магазине Утконос и получили результаты по метрике конверсий в покупку. Выберите метод оценки и оцените есть ли стат.значимые различия между конверсиями в двух группах при alpha = 5%. Дайте краткие рекомендации команде.  
Результаты:  
1)Число юзеров в группах , которые заходили на сайт в период эксперимента: n1 = 15550 и n2 = 15550.  
2)Число юзеров в группах , которые совершили хотя бы одну покупку за период эксперимента: n1 = 164 и n2 = 228.  
3)Конверсии : conv1 = 1.05% conv2 = 1.47%.  

2. Сравниваем метрику конверсия в покупку. Размер выборки - 10000 элементов в каждой группе . Какой статистический критерий тут лучше всего подойдёт и почему ?

### Задача 1.

In [12]:
import scipy.stats as stats
from math import sqrt

In [13]:
alpha = 0.05
n1 = 15550
n2 = 15550
conversion_n1 = 164
conversion_n2 = 228

In [14]:
# вычисляем значения конверсии в групах
conv1 = conversion_n1 / n1
print(conv1)
conv2 = conversion_n2 / n2
print(conv2)

0.010546623794212218
0.014662379421221864


К данной задаче небходимо применить Z критерий.

In [15]:
# вычисляем критические значения Z для двустороннего теста и уровня статистической значимости а = 0.05
stats.norm.ppf(1 - alpha / 2)

1.959963984540054

Критические значения z от -1.96 до 1.96

In [17]:
# расчитываем значение z критерия
p =  (conversion_n1 * conv1 + conversion_n2 * conv2) / (conversion_n1 + conversion_n2)
z = (conv1 - conv2) / sqrt(p * (1 - p) * (1/n1 + 1/n2))
z

-3.2110930329358367

Так как расчетное значение Z меньше критического (-3.211 < -1.96) принипаем альтернативную гипотезу. Упрощение формы заказа положительно повлияло на конверсию.

### Задача 2.

Сравниваем метрику конверсия в покупку. Размер выборки - 10000 элементов в каждой группе . Какой статистический критерий тут лучше всего подойдёт и почему ?

В данном случае обосновано применение Z критерия так как размер выборки 10000.